### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by mattheu**
Ran into a few smaller things to fix whilst implementing this in the handbook. Mostly minor tweaks to typography & colours.

### CODE REVIEW

File: src/styles/vendor/_prismjs.scss [hunk 1]
```diff
@@ -106,7 +79,7 @@ pre[class*="language-"] {
 .language-css .token.string,
 .style .token.string {
 	color: #a67f59;
-	background: hsla(0, 0%, 100%, .5);
+	// background: hsla(0, 0%, 100%, .5);
```
Comment (sambulance) [inline src/styles/vendor/_prismjs.scss]: Should this be removed if it isn't needed?

File: src/styles/base/_typography.scss [hunk 2]
```diff
@@ -165,6 +167,7 @@ pre[ class*="language-" ] {
 	border-radius: 2px;
 }
 
-::selection {
+::selection,
+pre::selection {
```
Comment (sambulance) [inline src/styles/base/_typography.scss]: Looks like the `pre::selection` stuff from prism.js was removed, so is this still needed? If it is, do we need `code::selected` too?

File: src/styles/base/_typography.scss [hunk 3]
```diff
@@ -156,6 +155,9 @@ pre {
 	padding: #{ $base-line-height * .5 } 15px;
 	display: block;
 	margin: #{ $base-line-height * .75 } 0;
+	max-width: 100%;
+	overflow: scroll;
```
Comment (sambulance) [inline src/styles/base/_typography.scss]: Should this be `auto` so we don't have scrollbars even when they aren't needed?

File: src/styles/base/_typography.scss (hunk 2 above)
Comment (mattheu) [inline src/styles/base/_typography.scss]: Actually this isn't necessary at all now as I just removed the styles from prism.scss. I'll remove it!

File: src/styles/base/_typography.scss (hunk 3 above)
Comment (mattheu) [inline src/styles/base/_typography.scss]: +1

**Comment by mattheu** [comment]
Pushed an update following your comments! Thanks.

**Pull request merged**